src/Layers/xrRenderGL/glDetailManager_VS.cpp

```diff
@@ -202,6 +207,15 @@ void CDetailManager::hw_Render_dump(CBackend& cmd_list, const Fvector4& consts,
                     u32 dwCNT_prims = dwBatch * Object.number_indices / 3;
                     //RCache.get_ConstantCache_Vertex().b_dirty				=	TRUE;
                     //RCache.get_ConstantCache_Vertex().get_array_f().dirty	(c_base,c_base+dwBatch*4);
+                    R_constant_load L = constArray->vs;
+                    if (constArray->destination & RC_dest_pixel) { L = constArray->ps; }
+                    if (constArray->destination & RC_dest_vertex) { L = constArray->vs; }
+                    if (constArray->destination & RC_dest_geometry) { L = constArray->gs; }
+                    if (constArray->destination & RC_dest_all) { L = constArray->pp; }
```

Comment:
1. We don't allow `if` one-liners. The body should always be on the new line.
2. And, in this case, parenthesis are not needed.
```suggestion
                    if (constArray->destination & RC_dest_pixel)
                        L = constArray->ps;
                    if (constArray->destination & RC_dest_vertex)
                        L = constArray->vs;
                    if (constArray->destination & RC_dest_geometry)
                        L = constArray->gs;
                    if (constArray->destination & RC_dest_all)
                        L = constArray->pp;
```